src/components/NavBar.js

```diff
@@ -2,15 +2,16 @@ import { NavLink } from 'react-router-dom';
 
 const Navbar = () => (
   <nav>
+    <p>Gabriel&apos;s bookstore</p>
     <ul>
       <li>
         <NavLink to="/books">
-          Books
+          BOOKS
         </NavLink>
       </li>
       <li>
         <NavLink to="/categories">
-          Categories
+          CATEGORIES
         </NavLink>
       </li>
     </ul>
```

Comment:
- [ ] It would be nice to implement the **_user icon_** at the top right corner of the books page. To highlight what @julie-ify noted, you don't have to implement the functionality because it is a placeholder. Remember that your design should be identical to that of the Zeplin template provided. Kindly implement it. 👍 

### Zeplin Design

![image](https://user-images.githubusercontent.com/77106962/179310448-1e768cd4-52df-4500-8515-1f75e81c7f25.png)

### Your Design

![image](https://user-images.githubusercontent.com/77106962/179310534-d4d1aa3b-58ef-4e3c-ac7a-398c24492d15.png)